### ISSUE TITLE
DOCUMENTATION_OPTIONS is not defined

### ISSUE THREAD

**Issue by stephenfin** [sphinx-doc/sphinx#4472]
Subject: Generated docs are showing console errors

### Problem
- After building docs the standard way (`tox -e docs`), I see the below errors in the console.

#### Procedure to reproduce the problem
```
tox -e docs
```

#### Error logs / results
```
Loading failed for the <script> with source “file:///home/sfinucan/Development/sphinx/sphinx/build/sphinx/html/_static/documentation_options.js”.    contents.html:12
jQuery.Deferred exception: DOCUMENTATION_OPTIONS is not defined
    initIndexTable@file:///home/sfinucan/Development/sphinx/sphinx/build/sphinx/html/_static/doctools.js:248:1
    init@file:///home/sfinucan/Development/sphinx/sphinx/build/sphinx/html/_static/doctools.js:150:5
    @file:///home/sfinucan/Development/sphinx/sphinx/build/sphinx/html/_static/doctools.js:310:3
    j@file:///home/sfinucan/Development/sphinx/sphinx/build/sphinx/html/_static/jquery.js:2:29997
    g/</k<@file:///home/sfinucan/Development/sphinx/sphinx/build/sphinx/html/_static/jquery.js:2:30313

ReferenceError: DOCUMENTATION_OPTIONS is not defined                                        doctools.js:248:1

initIndexTable    file:///home/sfinucan/Development/sphinx/sphinx/build/sphinx/html/_static/doctools.js:248:1
init              file:///home/sfinucan/Development/sphinx/sphinx/build/sphinx/html/_static/doctools.js:150:5
<anonymous>       file:///home/sfinucan/Development/sphinx/sphinx/build/sphinx/html/_static/doctools.js:310:3
j                 file:///home/sfinucan/Development/sphinx/sphinx/build/sphinx/html/_static/jquery.js:2:29997
g/</k<            file:///home/sfinucan/Development/sphinx/sphinx/build/sphinx/html/_static/jquery.js:2:30313
```

#### Expected results
There shouldn't be an issues.

### Reproducible project / your project
- N/A

### Environment info
- OS: Fedora 27 (Linux)
- Python version: python 2.7.14
- Sphinx version: commit 9a3f401c469ec6c16be32ce1bda11d266ba88d76
- <Extra tools: Firefox 57.0.4

**Comment by tk0miya** [comment]
That would be fixed by #4473. Could you try it please?

**Comment by tk0miya** [comment]
Fixed by #4473
